testplan/common/utils/interface.py

```diff
@@ -157,22 +20,20 @@ class MethodSignatureMismatch(Exception):
     pass
 
 
-def check_signature(func, args_list):
+def check_signature(
+    func: callable, args_list: Union[list, str]
+) -> Optional[bool]:
     """
     Checks if the given function's signature matches the given list of args
 
     :param func: function whose signature to check
-    :type func: ``callable``
     :param args_list: list of arg names to match as signature
-    :type args_list: ``list`` of ``str``
 
     :return: ``None`` or ``True``
```

Comment:
No need to write the return values, those are coming from `Optional[bool]` already. Also I think the thing here is actually just `bool` and the function's docstring needs a `raises` directive. Because it cannot come back void, it either raises or returns a bool.